docs/en/index.md

```diff
@@ -74,11 +74,7 @@ doesn't make much sense, like numeric values). More complex data model with rela
 
 The Translatable module uses the "Storage in language rows" approach.
 
-Alternatives:
-
- * Tractorcow's [Fluent Module](https://github.com/tractorcow/silverstripe-fluent)
- * UncleCheese's [TranslatableDataObject](http://www.leftandmain.com/silverstripe-tips/2012/04/03/translatabledataobject-insanely-simple-translation/) (open source)
- * Kreationsbyran's [Multilingual Module 2.0](http://www.kreationsbyran.se/blogg/multilingual-module-2-0-for-silverstripe-cms-3-0/) (commercial)
+An alternative is Tractorcow's [Fluent Module](https://github.com/tractorcow/silverstripe-fluent).
```

Comment:
It might be worth saying that this is the suggested replacement for SS4. What do you think @chillu?

Reply:
@raissanorth I think carry on with making this change for now and the docs can be adjusted again if a community dev picks up SS4 work on this module